Tests/ClientRuntimeTests/OrchestratorTests/OrchestratorTests.swift

```diff
@@ -167,20 +168,22 @@ class OrchestratorTests: XCTestCase {
     }
 
     class TraceExecuteRequest: ExecuteRequest {
```

Comment:
The behavior of this type is unchanged; I added a variable to allow for counting of retry attempts that is separate from the original allowed count.